lib/user-interface/react-app/src/pages/rag/create-workspace/create-workspace-opensearch.tsx

```diff
@@ -15,9 +15,10 @@ const nameRegex = /^[\w+_-]+$/;
 const defaults: OpenSearchWorkspaceCreateInput = {
   name: "",
   embeddingsModel: null,
+  crossEncodingEnabled: false,
   crossEncoderModel: null,
   languages: [{ value: "english", label: "English" }],
-  hybridSearch: true,
+  hybridSearch: false,
```

Comment:
should be true if cross encoders have been enabled in the configuration

Reply:
Here we are just defining the constant, the value from the configuration is read below at line 38.